### DISCUSSION TITLE
How to get User from TweetList

### DISCUSSION THREAD

**Discussion by nniiggeell** [redouane59/twittered#270] [Q&A]
Hi. how do I get the User object from a TweetList?
I have tried TweetList.getData().get(0).getUser().getName();
In this case, I am trying to get the name of the user that posted the first tweet in the list. However I notice that the TweetData class does not support this operation.
Is there a way around this? 

Thank you.

**Comment by redouane59** [comment]
Hi,
I think your problem is similar to [that one](https://github.com/redouane59/twittered/issues/250#issuecomment-874948476).
Unfortunately it's a limit of the Twitter API.
